notes/1.0.3/watch.md

```diff
@@ -0,0 +1,25 @@
+### Fixes with compatibility implications
+
+### Improvements
+
+- Add `sbt.watch.mode` system property to allow switching back to old polling behaviour for watch. See below for more details. [#3597][3597] by [@stringbean][@stringbean]
+
+### Bug fixes
+
+#### Alternative watch mode
+
+sbt 1.0.0 introduced a new mechanism for watching for source changes based on the NIO `WatchService` in Java 1.7. On
+some platforms (namely macOS and some versions of Linux) this has led to long delays before changes are picked up. An
+alternative `WatchService` for these platforms is planned for sbt 1.1.0 ([#3527][3527]), in the meantime an option to
+switch back to the old behavior has been added.
+
+The new `sbt.watch.mode` JVM flag has been added with the following supported values:
+
+- `nio` (default): use the NIO based `WatchService`.
```

Comment:
Let's default to polling if NIO is slower.

Reply:
NIO is only slower on the affect platforms.

Reply:
So could we revert back on Mac?

Reply:
Sure. Then we'll refine it when we fix the underlying issue.